makefile

```diff
@@ -675,6 +678,16 @@ style:
 	   echo "No source files were changed.";\
 	fi
 
+style-check:
```

Comment:
I suggest we handle this by introducing `code-style` script in #1219 (see https://mybitbucket.llnl.gov/projects/MFEM/repos/tests/browse/code-style)

Reply:
@tzanio @acfisher 
It seems to me that the test you mention actually applies astyle change.
However in this PR, @acfisher introduced an "astyle check" that tells if style is OK.

I would rather use this check, because I don’t think it would be easy to modify the code during the PR testing. However it’s the perfect place to tell the developers they should have check their code for style compliance.

Is that OK?